### DISCUSSION TITLE
Resource discovery configuration for Private DNS zone & private Link Services

### DISCUSSION THREAD

**Discussion by ayouba88** [tomkerkhove/promitor#1792] [Q&A] [answered]
Hi,

I am using Scraper - v2.1.0 + Resource Discovery - v0.2.0 ( can be upgraded if needed )
i am trying to confguire the following resources to be collected 

- Private DNS zone
- private Link Services

but i am not able to find the right configurations in order to add into the resource discovery yaml file 
are these resources supported at all ? and if yes, where can i find the right configurations ( the file in GitHub does not contains them )

Thanks

**Comment by tomkerkhove** [comment, marked as answer]
Private DNS & Link Services are not supported for resource discovery, unfortunately.

**Reply by ayouba88** [reply]
Thanks, i meantioned this on different discussion, but is flexible servers (postgres) isn't supported also by resourse discovery ?

**Reply by tomkerkhove** [reply]
You can use the generic scraper when using the declared metrics, but resource discovery will not work I'm afraid.

Azure Database for PostgreSQL is supported for resource discovery though.
https://docs.promitor.io/configuration/v2.x/metrics/postgresql
